index.html

```diff
@@ -1,28 +1,231 @@
+          </div>
+          <p class="mb-0">
+            This involves using JavaScript to modify the structure and content
+            of web pages, including adding, removing, and modifying HTML
+            elements.
+          </p>
+        </div>
+        <div class="program d-flex flex-row flex-md-column">
+          <div
+            class="col-4 d-flex flex-column justify-content-center align-items-center"
+          >
+            <i class="fa-brands fa-react"></i>
+            <h3 class="mb-0">Framework</h3>
+          </div>
+          <p class="mb-0">
+            learning popular JavaScript frameworks and libraries such as React,
+            Vue.js, and Angular, which help development of web applications.
+          </p>
+        </div>
+        <div class="program d-flex flex-row flex-md-column">
+          <div
+            class="col-4 d-flex flex-column justify-content-center align-items-center"
+          >
+            <i class="fa-sharp fa-solid fa-server"></i>
+            <h3 class="mb-0">ServerSide</h3>
+          </div>
+          <p class="mb-0">
+            Using JavaScript to build server-side applications using Node.js,
+            which allows JavaScript tofull-stack web applications.
+          </p>
+        </div>
+        <div class="program d-flex flex-row flex-md-column">
+          <div
+            class="col-4 d-flex flex-column justify-content-center align-items-center"
+          >
+            <i class="fa-solid fa-bug"></i>
+            <h3 class="mb-0">Debugging</h3>
+          </div>
+          <p class="mb-0">
+            It involves analyzing the code to determine the source of the
+            problem and making changes to fix it. Using debuggers or logging
+            tools to track the program's execution.
+          </p>
+        </div>
+        <div class="program d-flex flex-row flex-md-column">
+          <div
+            class="col-4 d-flex flex-column justify-content-center align-items-center"
+          >
+            <i class="fa-sharp fa-solid fa-diagram-project"></i>
+            <h3 class="mb-0">Projects</h3>
+          </div>
+          <p class="mb-0">
+            Completing hands-on projects to apply the skills learned throughout
+            the course and build a portfolio of web development projects.
+          </p>
+        </div>
+      </div>
+      <br />
+      <button class="enrol">Enroll now</button>
+      <h5>
+        And take the first step towards a rewarding career in web development!"
+      </h5>
+    </section>
 
+    <section class="speak" id="speaker">
+      <h2>Feature Professors</h2>
+      <hr class="w-50 text-light" />
+      <div class="speaker d-flex flex-column flex-md-row" id="prof"></div>
+      <button
+        class="d-md-none w-50 d-flex justify-content-center align-items-center"
+      >
+        <h3 class="mb-0">More</h3>
+        <i class="fa-solid fa-chevron-down text-danger"></i>
+      </button>
+    </section>
```

Comment:
- [ ] The same thing applies here. Kindly tailor your design in line with the guiding template. This includes color: background-color, text-color among others. Kindly walk your desing alongside guiding tamplate.

### Your Design
![your-feature](https://user-images.githubusercontent.com/26597432/225552822-ff2c3833-cd53-46e4-9b2f-85657b66a290.png)

### Template Design
![temp-features](https://user-images.githubusercontent.com/26597432/225552860-dcc7c7de-399d-4e0c-9aad-94440de946d5.png)